### PR TITLE
Add OpInfo for torch.bucketize

### PR DESCRIPTION
**Pull request by saketh-are**

### CODE REVIEW

File: torch/testing/_internal/common_methods_invocations.py [hunk 1]
```diff
@@ -9258,6 +9274,14 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
                #                                          ~~~~~~ <--- HERE
                DecorateInfo(unittest.skip("Skipped!"), 'TestJit', 'test_variant_consistency_jit'),
            )),
+    OpInfo('bucketize',
+           dtypes=all_types_and(torch.bfloat16),
+           dtypesIfCUDA=all_types(),
+           sample_inputs_func=sample_inputs_bucketize,
+           supports_autograd=False,
+           skips=(
+               DecorateInfo(unittest.skip("Skipped!"), 'TestJit', 'test_variant_consistency_jit'),
```
Comment (mruberry) [inline torch/testing/_internal/common_methods_invocations.py:9283]: Add a comment for why this test is skipped

Comment (mruberry) [reply torch/testing/_internal/common_methods_invocations.py:9283]: Might as well make this an expected failure, too?

**Comment by mruberry** [comment]
Eventually we should look at deprecation bucketize, too: https://github.com/pytorch/pytorch/issues/54588

**Review by mruberry — APPROVED** [review]

File: torch/testing/_internal/common_methods_invocations.py (hunk 1 above)
Comment (saketh-are) [reply torch/testing/_internal/common_methods_invocations.py:9284]: How do I make something an expected failure? I couldn't find an example elsewhere in OpInfos.

**Review by mruberry — APPROVED** [review]
Cool! Failures are unrelated

**Pull request closed without merging**